class/openshift4-monitoring.yml

```diff
@@ -1,4 +1,15 @@
 parameters:
+  openshift4_monitoring:
+    =_manifest_urls:
```

Comment:
I'm adding these here because they don't matter for the user of the component.

This basically implements a fragile poor man's conditional dependency fetching since there's no way to tell Kapitan to ignore dependencies whose upstream doesn't exist.